x-pack/plugins/index_management/public/application/sections/home/index_list/index_table/index_table.js

```diff
@@ -321,9 +353,7 @@ export class IndexTable extends Component {
             data-test-subj={`indexTableCell-${fieldName}`}
           >
             <div className={`euiTableCellContent indTable__cell--${fieldName}`}>
-              <span className="eui-textLeft">
-                {this.buildRowCell(fieldName, value, index, appServices)}
-              </span>
+              <span className="eui-textLeft">{this.buildRowCell(index, columnConfig)}</span>
```

Comment:
Not needed now, but in the future we may want to allow the `columnConfig` to optionally override this `eui-textLeft` class. I'm not sure how often designers would want different alignments for cell content, but in the past I've had to do right alignment for columns with numbers.

Reply:
Yes, we had some numbers columns right aligned as well. We can add to the column config in the future when needed.